lib/elasticity/multi_search.rb

```diff
@@ -1,5 +1,7 @@
 module Elasticity
   class MultiSearch
+    include Enumerable
```

Comment:
Any specific reason why this is a enumerable?

Also, is the `each` method necessary here? Seems like you're not using it and personally I'm not a big fan since this is not exactly a collection object.